content/documentation.rst

```diff
@@ -62,20 +62,71 @@ semicolons, rather than after the Nth column.
     long lines.
 
 
-Writing User's Guides
-=====================
+About the Documentation Sections
+================================
+
+**Introduction**
+
+High level introduction to the software.
+
+
+**Installation and Upgrade Instructions**
+
+System requirements, installation, installation validation/procedures, upgrade
+procedures, uninstallation instructions.
+
+
+**Getting Started with SFTPPlus**
+
+Prerequisite to this page is the installation section.
+
+Contains information about the initial, basic configurations that come with the
+installation package and first account configurations.
+
+
+**Configuration Instructions**
+
+Should contain information to the general configuration principle and
+references for each configuration option.
+
+This section focuses only on the **individual configuration options**.
+
+Samples and guides in this section should be aimed at the Local Manager GUI
+and the text file configuration.
+
+For example, the HTTP/HTTPS configuration page is only restricted to the
+configuration options available for this service.
+
+
+**Operation (under 'Usage Instructions')**
+
+Should only contain general principles of operation.
+
+The section can go into further detail about a particular feature or service
+beyond the configuration file and configuration file options.
+
+For example, the HTTP/HTTPS operations page goes into detail about what actions
+are available with this service, examples of usage and more.
+
+
+**User's Guides**
 
 Pages in the User's Guides are used to describe how a task can be performed by
 applying various configuration options.
 
-This section is also used for other frequent questions sent to Support / Sales
-that are not otherwise covered in the main documentation.
+This section is also used for other frequent questions sent to Support / Sales.
 
-User guides can also be written to the more general audience depending on the
-content. It is a good idea to specifically list out who the audience is.
+Can be written to the more general audience. 
```

Comment:
Also:

Examples are not exactly "fully functional" - as in, someone can't just copy and paste them.

Examples might 'look like' it is fully functional, in that they reflect real world scenarios.  For example, instead of '/path/to' it is 'accounting/library'.